packages/graphql/src/schemaTypes/objectTypes/gql-Migration.ts

```diff
@@ -40,15 +40,15 @@ export const Migration = objectType({
 
     t.nonNull.string('configBeforeCode', {
       description: 'contents of the cypress.json file before conversion',
-      resolve: () => {
-        return ``
+      resolve: (source, args, ctx) => {
```

Comment:
Does it make sense to have two endpoints for this? I wonder if we should group them, eg

```
query {
  migration {
    config {
     before
     after
    }
  }
}
```

Reply:
I don't have a strong opinion on this, either way it will be two endpoints. The way you suggested will have another object type with two endpoints.

Reply:
It means creating a new Object type for things that have their place in the migration fragment.
What code would it simplify?

Reply:
My thought was grouping it under a `migration` namespace would make it easier to understand in the future - eg if you see `codeBeforeConfig` in isolation, it's not entirely clear what this actually does or what it is used for.

We could add a `description` property explaining it, perhaps?